spec/src/main/asciidoc/converters.asciidoc

```diff
@@ -41,6 +41,7 @@ The following ``Converter``s are provided by MicroProfile Config by default:
 * `double`, `java.lang.Double`, and `java.util.OptionalDouble`; a dot '.' is used to separate the fractional digits
 * `char` and `java.lang.Character`
 * `java.lang.Class` based on the result of `Class.forName`
+* `java.lang.String`
```

Comment:
It is an automatic converter. No need to be mentioned here. I think the javadoc needs to be updated to have that line removed instead.